### PR TITLE
[2.7] update readme files for new components

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Branch | 2.7 |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #17997 |
| License | MIT |
| Doc PR |  |

This completes @javiereguiluz's great work from #17997 by adapting the readme files of components added after the release of Symfony 2.3.

### CODE REVIEW

**Comment by xabbuh** [comment]
Oh, we need to take into account the split of the Security component too.

Status: Needs Work

File: src/Symfony/Component/ExpressionLanguage/README.md [hunk 1]
```diff
@@ -2,42 +2,14 @@ ExpressionLanguage Component
...
-    };
-    $language->register('upper', $compiler, $evaluator);
-
-    echo $language->evaluate('"foo" ~ upper(foo)', array('foo' => 'bar'));
-    // would output fooBAR
-
-    echo $language->compile('"foo" ~ upper(foo)');
-    // would output ("foo" . strtoupper($foo))
+evaluate expressions. An expression is a one-liner that returns a value
+(mostly, but not limited to, Booleans).
```
Comment (sstok) [inline src/Symfony/Component/ExpressionLanguage/README.md]: `Booleans and strings` ?

Comment (xabbuh) [inline src/Symfony/Component/ExpressionLanguage/README.md]: I reused the introduction from the documentation: http://symfony.com/doc/current/components/expression_language/introduction.html

I think if we want to improve this (which doesn't seem to be the worst idea), we should make the change there first.

**Comment by xabbuh** [comment]
Security sub-components updated too.

Status: Needs Review

File: src/Symfony/Bridge/PhpUnit/README.md [hunk 2]
```diff
@@ -3,50 +3,11 @@ PHPUnit Bridge
...
-Deprecation notices in the **Remaining/Other** section need some thought.
-You have to decide either to:
-
- * update your code to not use deprecated interfaces anymore, thus gaining better
-   forward compatibility;
- * or move them to the **Legacy** section (by using one of the above way).
+Resources
+---------
+
+  * [Documentation](http://symfony.com/doc/current/components/phpunit_bridge.html)
```
Comment (javiereguiluz) [inline src/Symfony/Bridge/PhpUnit/README.md]: Please use `https://` in this link.

File: src/Symfony/Component/Asset/README.md [hunk 3]
```diff
@@ -1,166 +1,14 @@
...
 
 Resources
 ---------
 
-You can run the unit tests with the following command:
-
-    $ cd path/to/Symfony/Component/Asset/
-    $ composer update
-    $ phpunit
+  * [Documentation](http://symfony.com/doc/current/components/asset/introduction.html)
```
Comment (javiereguiluz) [inline src/Symfony/Component/Asset/README.md]: `https://` here too.

File: src/Symfony/Component/ExpressionLanguage/README.md (hunk 1 above)
Comment (javiereguiluz) [inline src/Symfony/Component/ExpressionLanguage/README.md]: I agree with @xabbuh. Let's copy the current description to move forward fast and let's improve it in the future.

File: src/Symfony/Component/ExpressionLanguage/README.md [hunk 4]
```diff
@@ -2,42 +2,14 @@ ExpressionLanguage Component
...
 
 Resources
 ---------
 
-You can run the unit tests with the following command:
-
-    $ cd path/to/Symfony/Component/ExpressionLanguage/
-    $ composer.phar install --dev
-    $ phpunit
+  * [Documentation](http://symfony.com/doc/current/components/expression_language/introduction.html)
```
Comment (javiereguiluz) [inline src/Symfony/Component/ExpressionLanguage/README.md]: `https://` please

File: src/Symfony/Component/VarDumper/README.md [hunk 5]
```diff
@@ -1,14 +1,15 @@
...
-references into account. More than being immune to infinite recursion
-problems, it allows dumping where references link to each other.
-It explores recursive structures using a breadth-first algorithm.
+Resources
+---------
 
-The component exposes all the parts involved in the different steps of
-cloning then dumping a PHP variable, while applying size limits and having
-specialized output formats and methods.
+  * [Documentation](http://symfony.com/doc/current/components/var_dumper/introduction.html)
```
Comment (javiereguiluz) [inline src/Symfony/Component/VarDumper/README.md]: `https://` in this link

**Comment by xabbuh** [comment]
@javiereguiluz Thanks for the heads up. Fixed all the links.

**Comment by javiereguiluz** [comment]
:+1:

Status: Reviewed

**Comment by nicolas-grekas** [comment]
Thank you @xabbuh.

**Pull request merged**